### ISSUE TITLE
Correct the indentation of help output

### ISSUE THREAD

**Issue by sandeepkanabar** [elastic/elasticsearch-support-diagnostics#171]
Currently when we run diagnostics.sh without any parameters, it outputs the following:

```
Usage: <main class> [options]
  Options:
        --accessLogs
       Use this option to collect access logs as well.
       Default: false
        --archivedLogs
       Get archived logs in addition to current ones if present - No value
       required, only the option.
       Default: false
        --bzip
       Set this option true to use bzip istead of gzip.
       Default: false
    -?, --help
       Help contents.
       Default: false
  * -h, --host
       Required field.  Hostname, IP Address, or localhost.  HTTP access must be
       enabled.
       Default: <empty string>
    -s, --ssl, --https
       Use SSL?  No value required, only the option.
       Default: false
        --interval
       Elapsed time in seconds between diagnostic runs when in repeating mode.
       Default: 30
```

The formatting i.e indentation of help output needs to be corrected

**Comment by archanid** [comment]
Looks like this was fixed in JCommander several years ago: https://github.com/cbeust/jcommander/commit/0dcc308f1c08b831376a3e153e2a1ed1a6166697#diff-575d24e57415fb9932798cdfcb0811f9R117

Are we just using an old version?

**Comment by nemonster** [comment]
Fixed in version 6.1, released same day.
